rest/model/admin.go

```diff
@@ -1419,3 +1415,35 @@ func (c *APITriggerConfig) ToService() (interface{}, error) {
 		GenerateTaskDistro: FromAPIString(c.GenerateTaskDistro),
 	}, nil
 }
+
+type APIJasperConfig struct {
+	BinaryName       APIString `json:"binary_name"`
```

Comment:
I would omitempty on these. At the very least the port so that it doesn't display as 0 by default

Reply:
Done.